extensions/algolia/fieldgroups/algolia-profile.example.1.json

```
@@ -0,0 +1,25 @@
+{
+  "algolia-profile": {
```

```suggestion
  "algoliaProfile": {
```
After making the changes suggested for algolia-profile.schema.json, the example will change as specified above.

---

I have updated the schemas with the suggested namespace.